packages/wrangler/src/index.tsx

```diff
@@ -1379,19 +1397,17 @@ export async function main(argv: string[]): Promise<void> {
 
             // -- snip, extract --
 
-            if (!args.local) {
-              const loggedIn = await loginOrRefreshIfRequired();
-              if (!loggedIn) {
-                // didn't login, let's just quit
-                console.log("Did not login, quitting...");
-                return;
-              }
+            const loggedIn = await loginOrRefreshIfRequired();
```

Comment:
why did you remove the args.local checks here?

Reply:
Two reasons: 
- they are redundant due to the error being thrown earlier in the function if `args.local` is true
- if this block is wrapped in an if statement then TS is unable to tell that `config.account_id` is not undefined in the `fetchResult()` call below.